OrbitCore/TracepointEventBuffer.h

```diff
@@ -24,7 +24,10 @@ class TracepointEventBuffer {
   void AddTracepointEventAndMapToThreads(uint64_t time, uint64_t tracepoint_hash,
                                          int32_t thread_id);
 
-  std::map<int32_t, std::map<uint64_t, orbit_client_protos::TracepointEventInfo> >
+  [[nodiscard]] const std::map<uint64_t, orbit_client_protos::TracepointEventInfo>&
```

Comment:
I think you can delete "GetTracepointEventsInTimeRangeForThreadId" from class TracepointEventBuffer. This looks like it was supposed to do the stuff you are now doing in UpdatePrimitives on the map directly.

Reply:
Sure. You are right. Thank you!

Reply:
Yes. You are right. Thank you!